test/DateTimeTest.php

```diff
@@ -544,16 +558,19 @@ public function testDiff()
         $this->assertEquals('-P31Y10M1DT0H0M5.000001S', $dt2->diff($dt1)->format('%RP%yY%mM%dDT%hH%iM%sS'));
     }
 
-    public function testDiffInvalidArgument() {
-        $this->setExpectedException('\InvalidArgumentException');
+    public function testDiffInvalidArgument(): void
+    {
+        $this->expectException('InvalidArgumentException');
```

Comment:
```suggestion
        $this->expectException(InvalidArgumentException::class);
```

For this, you will need to add also the `use InvalidArgumentException` clause in the top of the file